share/mrdocs/addons/generator/asciidoc/layouts/overload-set.adoc.hbs

```diff
@@ -3,16 +3,66 @@
 
 = {{#if symbol.name}}{{>nested-name-specifier symbol=symbol.parent includeNamespace=true}}{{symbol.name}}{{else}}Unnamed overload set{{/if}}
 
-== Members
+{{#if symbol.members.[0]}}
+  
+{{#if symbol.members.[0].doc.brief}}
 
-{{#each symbol.members as | member |}}
+{{symbol.members.[0].doc.brief}}
+
+{{/if}}
 
-{{member.doc.brief}}
+== Synopsis
+  
+{{#each symbol.members as | member |}}
 
 [source,cpp,subs="verbatim,macros,-callouts"]
 ----
 {{> (concat 'signature' '/' (lookup member 'kind')) symbol=member link=member}};
-  xref:{{ref}}[pass:c,q,m[*_» more..._*]]
 ----
+{{/each}}
+  
+{{#if symbol.members.[0].doc.description}}
+== Description
+
+{{symbol.members.[0].doc.description}}
+{{/if}}
+
+
+{{#with (flattenUnique symbol.members "doc.params" "name") as |allParams|}}
```

Comment:
can't we achieve the same as `flattenUnique` as a combination of two existing operations?

Reply:
> can't we achieve the same as `flattenUnique` as a combination of two existing operations?

I tried combining several existing operations before implementing `flattenUnique`, but I was not successful in achieving the desired result. Therefore, I decided to implement this new function.